packages/app/src/app/pages/Dashboard/Components/Selection/index.tsx

```diff
@@ -413,6 +414,7 @@ export const SelectionProvider: React.FC<SelectionProviderProps> = ({
 
   const onDragStart = React.useCallback(
     (event: React.MouseEvent<HTMLDivElement>, itemId: string) => {
+      analytics.track('Dashboard - On drag start');
```

Comment:
We could update this with more insights by passing what type of folder / sandbox is dragged?